include/NDRegisterAccessorAbstractor.h

```diff
@@ -48,6 +48,10 @@ namespace ChimeraTK {
      */
     const NDRegisterAccessorAbstractor& operator=(const NDRegisterAccessorAbstractor& rightHandSide) const = delete;
 
+    boost::shared_ptr<NDRegisterAccessor<UserType>> getImpl();
+
+    using value_type = UserType;
```

Comment:
value_type is not our coding style. Will the linter complain? What is it used for?

Reply:
Sorry, this should not have been committed, I removed it.